.eslintrc.cjs

```diff
@@ -1,102 +0,0 @@
-module.exports = {
-  settings: {},
-  env: {
-    node: true,
-  },
-  extends: [
-    "eslint:recommended",
-    "plugin:import/recommended",
-    "plugin:@typescript-eslint/eslint-recommended",
-    "plugin:@typescript-eslint/recommended",
-    "plugin:@typescript-eslint/recommended-requiring-type-checking",
-    "prettier",
-    "plugin:jsdoc/recommended-typescript",
```

Comment:
the eslint-config-neolution doesn't contain the jsdoc, here it should continue to use it by extending the eslint-config-neolution (if it's not planned to add it directly in the eslint-config-neolution )

Reply:
Ignore this comment, I now noticed that the jsdoc is present in eslint-config-neolution 👍🏼

Reply:
i added it in the meantime, just wasnt able to update all packages and reply to all comments 😄